src/segments/zvm.go

```diff
@@ -0,0 +1,74 @@
+package segments
+
+import (
+	"strings"
+
+	"github.com/jandedobbeleer/oh-my-posh/src/properties"
+	"github.com/jandedobbeleer/oh-my-posh/src/runtime"
+)
+
+type Zvm struct {
```

Comment:
this needs to inherit from `base`, that way you don't need to specify the missing functions.

Reply:
Fixed